src/container/srv_target.c

```diff
@@ -2603,6 +2591,13 @@ cont_child_prop_update(void *data)
 		return rc;
 	}
 	D_ASSERT(child != NULL);
+	if (child->sc_stopping || child->sc_destroying) {
```

Comment:
This looks unnecessary for me, cont_child_lookup() will return ENOENT if it was stopped